Makefile

```diff
@@ -32,6 +32,7 @@ test:
 install-deps-new:
 	virtualenv -p /usr/bin/python2.7 tut; \
 	. tut/bin/activate; \
+	pip install --upgrade setuptools; \
```

Comment:
Is this just because of the warning or is there any other reason why this must be updated? As far as I know, the version setup tools used in any image is based on the shipped version with the distribution.

Reply:
The install-deps-new fails because the old setuptools fails with folowing error when attempting to install pytest package:

    error in pytest setup command: Invalid environment marker: python_version<"3.0"